spec/models/druid_version_zip_spec.rb

```diff
@@ -15,6 +15,14 @@
     it 'returns a tree path-based key' do
       expect(dvz.s3_key).to eq 'bj/102/hs/9687/bj102hs9687.v0001.zip'
     end
+
+    context 'version is greater than 1' do
+      let(:version) { 2 }
+
+      it 'uses the right version in the path' do
+        expect(dvz.s3_key).to eq 'bj/102/hs/9687/bj102hs9687.v0002.zip'
```

Comment:
Is this just bulletproofing?  Like this test would not fail previous to the above change, right?

Reply:
correct